From: xaizek <xaizek@openmailbox.org>
Date: Sun, 26 Oct 2014 11:37:02 +0100
Forwarded: yes
Description: Fix buffer overflow in tests to fix building on arm

---
 tests/misc/strchar2str.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/misc/strchar2str.c b/tests/misc/strchar2str.c
index abbfbe7..3dd44c5 100644
--- a/tests/misc/strchar2str.c
+++ b/tests/misc/strchar2str.c
@@ -39,7 +39,7 @@ test_tabulation_is_expanded_properly(void)
 	for(pos = 0; pos < cfg.tab_stop; pos++)
 	{
 		size_t screen_width;
-		char out_buf[cfg.tab_stop];
+		char out_buf[cfg.tab_stop + 1];
 		const char *const str = strchar2str("\t", pos, &screen_width);
 		const int space_count = cfg.tab_stop - pos%cfg.tab_stop;
 		memset(out_buf, ' ', space_count);
